scripts/shell_completions/zsh/_borg

```diff
@@ -1148,7 +1148,7 @@ _borg_format_keys() {
   local repo_or_arch=${(Q)1}
 
   local -a keys=( NEWLINE NL NUL SPACE TAB CR LF )
-  local -a repository_keys=( archive name barchive comment bcomment id start time end hostname username )
+  local -a repository_keys=( archive name barchive comment bcomment id start time end command_line hostname username )
```

Comment:
ok, so you fixed zsh - are there other completions to fix also?

Reply:
That was not manual. It was some magic and I trusted it.

Reply:
Oh? OK...